### PR TITLE
Keep line width to 79 chars

### PR DESCRIPTION
**Pull request by jamesray1**
To be able to see two files on one screen. I've changed tab widths to 4 chars.

### CODE REVIEW

File: pb/rpc.proto [hunk 1]
```diff
@@ -11,45 +11,44 @@ message RPC {
...
-	optional ControlMessage control = 3;
+    optional ControlMessage control = 3;
 }
 
 message Message {
 	optional bytes from = 1;
 	optional bytes data = 2;
 	optional bytes seqno = 3;
 	repeated string topicIDs = 4;
-	optional bytes signature = 5;
```
Comment (lanzafame) [inline pb/rpc.proto:22]: Were these two lines meant to be deleted?

**Review by lanzafame — COMMENTED** [review]
There is now mixed indentation, and the comments are harder to grok because of the line break, plus if anyone were to fix the indenting, i.e. `gg=G` in vim, then the comments are no longer obviously associated.

File: pb/rpc.proto (hunk 1 above)
Comment (jamesray1) [reply pb/rpc.proto:22]: Oh, no, looks like those lines have been added some time recently, I was editing from an older file. :S

**Comment by jamesray1** [comment]
@lanzafame that's weird, the rendering on GitHub is different on VS Code and in the GitHub editor, where the indentation is lined of comments are lined up.

**Comment by lanzafame** [comment]
Yeah that's because Github and VS Code will render tabs at a different size, see image for how tabs are now used for TopicDescriptor to the end of the file:

Also because of the tabs between the field and the comments more comment than previously necessary needs to be on the next line down. It is almost not worth it in my opinion.

File: pb/rpc.proto [hunk 2]
```diff
@@ -11,45 +11,44 @@ message RPC {
...
-	optional bytes key = 6;
```
Comment (vyzo) [inline pb/rpc.proto:23]: Please don't remove those!

**Review by vyzo — CHANGES_REQUESTED** [review]
Do we really care to keep things in 79 chars?
We are not using ttys anymore and most people edit in much wider screen areas.

Also, the indentation for the comments now looks very strange.

**Comment by vyzo** [comment]
I would like to add that I am very much against this change; we don't have any char limitation anywhere else and it really doesn't make much sense in 2018.

**Comment by lanzafame** [comment]
At this point, the readability is being sacrificed for 'ideal' of 79 char limit. Especially, when this can be solved in editor-space by a soft wrap toggle hotkey or shrinking the text size for the time it takes to read the 3 words that go over the 79 char limit.

**Comment by jamesray1** [comment]
> Especially, when this can be solved in editor-space by a soft wrap toggle hotkey or shrinking the text size for the time it takes to read the 3 words that go over the 79 char limit.

I wasn't aware of the former in VS code, but looked into it and I see that you can use Alt+Z. I tried to shrink and expand selected text but that didn't work. I didn't think zoom in and out was a feature in VS Code, but I checked again and you can use Ctrl+-/+.

**Comment by jamesray1** [comment]
Having to zoom out (and squint) or keep scrolling across isn't ideal, however.

**Comment by lanzafame** [comment]
Agreed, hence why I suggested a soft wrap toggle first as it doesn't
require scrolling across or squinting.

**Comment by jamesray1** [comment]
Right, whoops, momentarily forgot about that! Thanks!

**Pull request closed without merging**